src/sentry/tasks/post_process.py

```diff
@@ -120,6 +122,24 @@ def _capture_group_stats(job: PostProcessJob) -> None:
     metrics.incr("events.unique", tags=tags, skip_internal=False)
 
 
+def should_issue_owners_ratelimit(
```

Comment:
I think this could be a generic utility function. Do we have something already in one of our utils modules?

Do we have tests for this?